_meta/fields.common.yml

```diff
@@ -30,6 +30,67 @@
         Any arbitrary contextual information regarding the event, captured by the agent, optionally provided by the user.
       dynamic: false
       fields:
+        - name: custom
+          type: object
+          enabled: false
+          indexed: false
+          analyzed: false
+          searchable: false
+          aggregatable: false
+
+        - name: db
+          type: group
+          fields:
+            - name: instance
+              type: keyword
+              index: false
+              analyzed: false
+              searchable: false
+              aggregatable: false
+
+            - name: statement
+              type: keyword
+              index: false
+              analyzed: false
+              searchable: false
+              aggregatable: false
+
+            - name: type
+              type: keyword
+              index: false
+              analyzed: false
+              searchable: false
+              aggregatable: false
+
+            - name: user
+              type: keyword
+              index: false
+              analyzed: false
+              searchable: false
+              aggregatable: false
+
+        - name: http
+          type: group
+          dynamic: false
+          fields:
+            - name: method
+              type: keyword
+              index: false
+              aggregatable: false
+              searchable: false
+              doc_values: false
+
+            - name: status_code
```

Comment:
This is an indexed field, same datatype as `context.response.status_code` that is aliased to `http.response.status_code` now. I think this should also be mapped to the same field, by copying it over instead of aliasing.

Reply:
I convinced myself that this field is metadata on a span and might actually be confusing if it's in `http.response.status_code`, as it's the response code from calling a remote service vs the response code we return from the traced service.

If you would still like to proceed, are you saying that in:
6.6 - put this value in `context.response.status_code` and `context.http.status_code`
7.0 - write it to `http.response.status_code`

Reply:
I can see the ambiguity here. This is to track the response of an external call performed by the monitored app, correct?

I think I'd be inclined to still put it in the ECS field. After all, these events will mostly be looked at in the context of this one event stream. Might as well benefit from the streamlined field names.

But if you see a problem with this, we can hold off on the migration of this field.

Reply:
> This is to track the response of an external call performed by the monitored app, correct?
correct

I don't have philosophical objections to this plan - my concern is mostly a practical issue of this being our only 1:many field.  I agree that in 7.0.0 we can write this to `http.response.status_code`, after all. an APM event stream in the distributed tracing case is composed of multiple transactions that will each bring a possibly different `http.response.status_code` value.  It's also not a big effort to write this to both places for spans, I just want to be sure we want that in 6.6 because I'm not sure what the benefit of that is, since we can't make that work for pre-6.6 indices.

Reply:
ok, I am good with leaving as is for 6.x and reconsider for 7.0